tests/k8s/test_k8s.py

```diff
@@ -957,8 +957,8 @@ def test_flow_multiple_protocols_gateway(logger, docker_images, tmpdir):
         protocol=['http', 'grpc'],
     )
 
-    dump_path = os.path.join(str(tmpdir), 'k8s-flow-custom-gateway.yml')
-    namespace = 'flow-custom-gateway'.lower()
+    dump_path = os.path.join(str(tmpdir), 'k8s-flow-multiprotocol-gateway.yml')
+    namespace = 'flow-multiprotocol-gateway'.lower()
```

Comment:
why do we need these changes?

Reply:
the whole test is named `test_flow_multiple_protocols_gateway` while `test_flow_with_custom_gateway` is unchanged